docs/content/en/docs/getting-started/observability/_index.md

```diff
@@ -1,6 +1,281 @@
+why a deployment took so long or why it failed.
+
+## Using this exercise
+
+This exercise shows how to standardize access
+to the observability data for your cluster.
+It is based on the
+[simplenode-dev](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd)
+example.
+You can clone that repo to access it locally
+or just look at it for examples
+as you implement the functionality "from scratch"
+on your local Kubernetes deployment cluster.
+The
+[README](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/setup/observability/README.md)
+file for that repo contains useful information.
+
+Two videos are available
+to walk you through this exercise if you prefer:
+
+- [Introducing Keptn Lifecycle Toolkit](https://youtu.be/449HAFYkUlY)
+- [Use SLOs and get DORA the Native K8s way!](https://www.youtube.com/watch?v=zeEC0475SOU)
+
+In the
+[Getting started with Keptn metrics](../metrics)
+exercise, you learn how to define and use Keptn metrics.
+You may want to complete that exercise before doing this exercise
+although that is not required.
+
+This exercise shows how to standardize access
+to the observability data for your cluster.
+The steps are:
+
+1. [Install and enable]( #install-and-enable-klt)
+   the Lifecycle Toolkit on your cluster
+1. [Integrate the Lifecycle Toolkit with your applications](#integrate-the-lifecycle-toolkit-with-your-applications)
+1. [DORA metrics](#dora-metrics)
+1. [Using OpenTelemetry](#using-opentelemetry)
+1. [Keptn metrics](#keptn-metrics)
+1. [View the results](#view-the-results)
+
+## Install and enable KLT
+
+To install and enable the Keptn Lifecycle Toolkit on your cluster:
+
+1. Be sure that your cluster includes the components discussed in
+   [Prepare your cluster for KLT](../../install/k8s.md/#prepare-your-cluster-for-klt)
+1. Follow the instructions in
+   [Install the Keptn Lifecycle Toolkit](../../install/install.md/#use-helm-chart)
+   to install KLT on your cluster using the Helm chart
+
+   If you installed KLT on your cluster for the
+   [Getting started with Keptn metrics](../metrics)
+   exercise, you do not need to re-install it for this exercise.
+   However, if you only installed the `metrics-operator` for that exercise,
+   you now need to install the full KLT.
+
+1. Follow the instructions in
+   [Enable KLT for your cluster](../../install/install.md/#enable-klt-for-your-cluster)
+   to enable KLT on your cluster
+   by annotating the `Namespace` resource..
+   See the
+   [simplenode-dev-ns.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/simplenode-dev-ns.yaml)
+   file for an example
+
+1. Run the following command to ensure that your Kuberetes cluster
+   is ready to complete this exercise:
+
+   ```shell
+   kubectl get pods -n keptn-lifecycle-toolkit-system
+   ```
+
+   You should see pods for the following components:
+   - certificate-operator (or another cert manager)
+   - lifecycle-operator
+   - scheduler
+   - otel-collector
```

Comment:
We don't install this and users might have it in a different namespace. I would remove from the list

Reply:
done

Reply:
done